Please investigate the following compiler warnings
```
/home/omnesflo/Antares_Simulator/src/tests/end-to-end/binding_constraints/test_binding_constraints.cpp:2: warning: "BOOST_TEST_DYN_LINK" redefined
    2 | #define BOOST_TEST_DYN_LINK
      | 
<command-line>: note: this is the location of the previous definition
/home/omnesflo/Antares_Simulator/src/tests/end-to-end/simple_study/simple-study.cpp:2: warning: "BOOST_TEST_DYN_LINK" redefined
    2 | #define BOOST_TEST_DYN_LINK
      | 
<command-line>: note: this is the location of the previous definition
```

--- a/src/tests/end-to-end/simple_study/CMakeLists.txt
+++ b/src/tests/end-to-end/simple_study/CMakeLists.txt
@@ -31,8 +31,6 @@ target_include_directories(tests-simple-study
 							${CMAKE_CURRENT_SOURCE_DIR}/../utils
 							)
 
-target_compile_definitions(tests-simple-study PRIVATE "BOOST_TEST_DYN_LINK=1")
-
 add_test(NAME end-to-end-simple-study COMMAND tests-simple-study)
 set_property(TEST end-to-end-simple-study PROPERTY LABELS end-to-end)
 set_target_properties(tests-simple-study PROPERTIES FOLDER Unit-tests/end_to_end)

--- a/src/tests/end-to-end/utils/CMakeLists.txt
+++ b/src/tests/end-to-end/utils/CMakeLists.txt
@@ -18,5 +18,4 @@ target_include_directories(test_utils
 
 set_target_properties(test_utils PROPERTIES FOLDER Unit-tests/end_to_end)
 
-target_compile_definitions(test_utils PUBLIC "BOOST_TEST_DYN_LINK=1")
 target_include_directories(test_utils PUBLIC ${Boost_INCLUDE_DIRS})

